### PR TITLE
Initialize acoustic perception score

### PR DESCRIPTION
**Pull request by caguero**
This is a bug fix for the acoustic perception task. While running phase 2, I noticed an issue with the scoring. The score should be equal to the elapsed time needed to reach the pinger. However, if the team doesn't reach the pinger, the plugin was reporting a score of `0`. Keep in mind that the lower the score the better. Instead, in that case the score should equal to the total running time (e.g. `300`).

How to test it?

Run this task and let it timeout while watching the score.

```
ros2 launch vrx_gz competition.launch.py world:=practice_2023_acoustic_perception0_task
```

And then:

```
ros2 topic echo /vrx/task/info
```

### CODE REVIEW

File: vrx_gz/src/AcousticPerceptionScoringPlugin.cc [hunk 1]
```diff
@@ -86,6 +86,8 @@ void AcousticPerceptionScoringPlugin::Configure(const sim::Entity &_entity,
       gzerr << "Error creating visual marker" << std::endl;
     }
   }
+
+  this->SetScore(this->remainingTime().count());
```
Comment (j-herman) [inline vrx_gz/src/AcousticPerceptionScoringPlugin.cc:90]:
```suggestion
  this->SetScore(this->RemainingTime().count());
```

**Review by j-herman — CHANGES_REQUESTED** [review]

**Comment by j-herman** [comment]
@caguero This doesn't seem to be working as intended.  The score stays at zero when I run the acoustic perception task.
I didn't dig in too far, but it looks like SetScore isn't available until we get to the running state, and the remaining time stays at zero for the first couple of task message updates.  Maybe use the timeout score?

**Comment by caguero** [comment]
> 6ac1771

Sorry, wrong code. It should work now.

**Review by j-herman — APPROVED** [review]
Works as described!

**Pull request merged**